docs/api/commands/wait.mdx

```diff
@@ -16,6 +16,47 @@ cy.wait(alias, options)
 cy.wait(aliases, options)
 ```
 
+```typescript
+// Specifying request and response types for aliased intercepts
+type UserReq = {}
+type UserRes = {}
+type ActivityReq = {}
+type ActivityRes = {}
+
+cy.intercept('/users/*').as('getUsers')
+cy.intercept('/activities/*').as('getActivities')
+
+// As templated types:
+cy.wait<CustomRequest, CustomResponse>('@getUsers').then(
```

Comment:
The types here will need to match the ones above (and in the callback body):
`cy.wait<UserReq, UserRes>(...`